### DISCUSSION TITLE
No such BSSID available.

### DISCUSSION THREAD

**Discussion by Londopy** [aircrack-ng/aircrack-ng#2362] [General]
$ sudo aireplay-ng --ignore-negative-one -0 0 -a B0:E4:D5:89:E8:26 wlan0
 
ioctl(SIOCSIWMODE) failed: Device or resource busy
17:32:50  Waiting for beacon frame (BSSID: B0:E4:D5:89:E8:26) on channel -1
17:33:00  No such BSSID available.

**Comment by Mister-X-** [comment]
You need to provide more details to diagnose the issue.

We'll need the usual info:
- what is the output of airmon-ng --verbose?
- Is there anything in dmesg that indicates an issue? Don't paste the whole output there, it's just a question for you, to check in your dmesg/journalctl to see if there are any issues related to wlan0 or that driver. All we need is a yes or no at this point

Just FYI, `--ignore-negative-one` isn't needed. Why are you using it?

And please, use markdown for readability.
